File: H5/H5/Resources/.generated/System/MissingMethodException.js
    H5.define("System.MissingMethodException", {
        inherits: [System.Exception],
        ctors: {
            ctor: function () {
                this.$initialize();
                System.Exception.ctor.call(this, "Attempted to access a missing method.");
            },
            $ctor1: function (message) {
                this.$initialize();
                System.Exception.ctor.call(this, message);
            },
            $ctor2: function (message, inner) {
                this.$initialize();
                System.Exception.ctor.call(this, message, inner);
            },
            $ctor3: function (className, methodName) {
                this.$initialize();
                System.Exception.ctor.call(this, (className || "") + "." + (methodName || "") + " Due to: Attempted to access a missing member.");
            }
        }
    });

File: H5/H5/System/MissingMethodException.cs
using System.Collections.Generic;

namespace System
{
    public class MissingMethodException : Exception
    {
        public MissingMethodException()
            : base("Attempted to access a missing method.")
        {
        }

        public MissingMethodException(string message)
            : base(message)
        {
        }

        public MissingMethodException(string message, Exception inner)
            : base(message, inner)
        {
        }

        public MissingMethodException(string className, string methodName) : base (className + "." + methodName + " Due to: Attempted to access a missing member.")
        {
        }
    }
}